app/browser/api/ledger.js

```diff
@@ -1340,7 +1338,10 @@ const enable = (state, paymentsEnabled) => {
 
     togglePromotionTimeoutId = setTimeout(() => {
       checkPromotions()
-    }, random.randomInt({min: 10 * ledgerUtil.milliseconds.second, max: 15 * ledgerUtil.milliseconds.second}))
+    }, process.env.LEDGER_ENVIRONMENT === 'staging'
+      ? random.randomInt({min: 10 * ledgerUtil.milliseconds.second, max: 15 * ledgerUtil.milliseconds.second})
+      : random.randomInt({min: 60 * ledgerUtil.milliseconds.second, max: 90 * ledgerUtil.milliseconds.second})
```

Comment:
I think this is not needed any more, because we trigger check only after init is done, so we can stay with 10-15sec

cc @bbondy

Reply:
I prefer the longer one but if there are strong objections or risks of breaking things, I'll leave it to you guys. Thanks for checking.

Reply:
Your call @NejcZdovc

Reply:
Done.